src/Donations/Repositories/DonationRepository.php

```diff
@@ -276,6 +277,8 @@ public function update(Donation $donation)
 
         DB::query('COMMIT');
 
+        clean_post_cache($donation->id);
+
```

Comment:
@glaubersilva instead of adding this directly here, can you please move this into a new legacy listener called `ClearDonationPostCache`.  In `src/Donations/ServiceProvider.php` you'll find where all the legacy listeners are being called from our new givewp hooks.  The idea is that we can use actions like `givewp_donation_updated` to support all backwards compatibility in an isolated way instead of adding directly to the repository methods.

Reply:
I did some commits to implementing it. ;-)